CustomReporter.php

```diff
@@ -3,20 +3,20 @@
 class CustomReporterPlugin extends MantisPlugin {
 
 	function register() {
-		$this->name = lang_get( 'plugin_customreporter_title' );
-		$this->description = lang_get( 'plugin_customreporter_description' );
+		$this->name = plugin_lang_get( 'title' );
+		$this->description = plugin_lang_get( 'description' );
 		$this->page = 'config';
-		$this->version = '1.03';
-		$this->requires = array( 'MantisCore' => '1.2.0', );
-		$this->author = 'Carlos Proensa, Cas Nuy, Damien Regad';
+		$this->version = '1.04';
+		$this->requires = array( 'MantisCore' => '<2.1.99', );
```

Comment:
Thanks a lot for your work!

I have tested your changes in version 2.2.1. Therefore I had to change this line to <2.2.99. With this little change it is perfectly working.
Maybe this requirement should be changed to `'MantisCore' => '2.1.0'` since it is very likely that it will work also in further versions and it avoids the need of changing this line with every release.